storage/objects.go

```diff
@@ -375,6 +485,211 @@ func createDirectory(c ObjectsClient, ctx context.Context, absPath _AbsCleanPath
 	return nil
 }
 
+func abortMpu(c ObjectsClient, ctx context.Context, input *AbortMpuInput) error {
+	reqInput := client.RequestInput{
+		Method:  http.MethodPost,
+		Path:    input.PartsDirectoryPath + "/abort",
+		Headers: &http.Header{},
+		Body:    nil,
+	}
+	respBody, _, err := c.client.ExecuteRequestStorage(ctx, reqInput)
+	if err != nil {
+		return errors.Wrap(err, "unable to abort mpu")
+	}
+
+	if respBody != nil {
+		defer respBody.Close()
+	}
+
+	return nil
+}
+
+func commitMpu(c ObjectsClient, ctx context.Context, input *CommitMpuInput) error {
+	headers := &http.Header{}
+	for key, value := range input.Headers {
+		headers.Set(key, value)
+	}
+
+	id := input.Id
+	idLength := len(id)
+	prefixLen, err := strconv.Atoi(id[idLength-1 : idLength])
+	if err != nil {
+		return errors.Wrap(err, "unable to commit mpu")
```

Comment:
Maybe tweak the error string "unable to commit mpu" to be "invalid mpu prefix length" ?

Reply:
Also - a comment on why this is here - I'd be confused if I hadn't heard you already talk about this special byte.

Reply:
👍